### PR TITLE
more additonal domains

### PR DESCRIPTION
**Pull request by tendervittles**
These domains are equivalent on bitwarden.

### CODE REVIEW

**Comment by Carlgo11** [comment]
Do you have any more?

**Comment by tendervittles** [comment]
That is it for me.

**Review by Carlgo11 — APPROVED** [review]

File: entries/f/facebook.com.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
   "Facebook": {
     "domain": "facebook.com",
+    "additional-domains": [
+      "messenger.com"
+    ],
```
Comment (phallobst) [inline entries/f/facebook.com.json:6]: I'm not quite sure about this one. It seems to deserve a separate entry in `communication`.

Comment (Carlgo11) [reply entries/f/facebook.com.json:6]: I'm not sure I agree here. As Messanger is a part of the Facebook experience and can't be severed from it I don't think a separate entry is needed for it. It's basically the PM feature of Facebook like the DMs on Twitter or private messages on Discord.

Comment (person1234-ux) [reply entries/f/facebook.com.json:6]: I believe this should just be give added as an addition domain. As said before it's basically like the pm feature on twitter.

**Pull request merged**